### DISCUSSION TITLE
Transparency

### DISCUSSION THREAD

**Discussion by sOmEoNe2lOvEgIt** [felixse/FluentTerminal#1014] [Q&A] [answered]
Hello!
I've been using fluent terminal for a while now and i've loved the fact that u can make it transparent! The only thing is that this transparency has a pretty anoying limitation, the whole transparent background fades to black when you change desktops or when you click out of the window... Would there be any way to "fix" that or is it a limitation due to windows?

**Comment by hanskokx** [comment, marked as answer]
This is a limitation in Windows. The Windows devs have chosen to limit the number of active transparency effects for performance reasons.

—
Hans Kokx
On 7/18/2022 5:03:58 AM, sOmEoNe2lOvE ***@***.***> wrote:
Hello!
I've been using fluent terminal for a while now and i've loved the fact that u can make it transparent! The only thing is that this transparency has a pretty anoying limitation, the whole transparent background fades to black when you change desktops or when you click out of the window... Would there be any way to "fix" that or is it a limitation due to windows?
—
Reply to this email directly, view it on GitHub [https://github.com/felixse/FluentTerminal/discussions/1014], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AAOSY36V3L3NQSGIZ6A5NSDVUUMX3ANCNFSM533P4UMA].
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[0f5b05c8-ea93-475b-a5b7-860fae747c28]

**Comment by LuminousFellow** [comment]
When the window is inactive, it looks great, but when I click in the window, it goes grey and is much harder to read.

How can I reverse this behavior or just be rid of the grey coloring altogether?

**Reply by LuminousFellow** [reply]
No reply, so I went to "Settings" -> "Terminal" and turned off "Use acrylic background".
All good now.
